react/index.js

```diff
@@ -366,9 +382,21 @@ class MiniCart extends Component {
                   </span>
                 )}
               </span>
-              {iconLabel && (
-                <span className={iconLabelClasses}>{iconLabel}</span>
-              )}
+              <span className="flex flex-column items-start">
```

Comment:
Wrap this whole thing in a condition to only create this element if `iconLabel || (showPrice && totalPrice > 0)`.

Reply:
and perhaps create something like `const isPriceVisible = showPrice && totalPrice > 0`, and use it both here and down below